ocaml/xapi/repository_helpers.ml

```diff
@@ -638,6 +638,19 @@ module UpdateInfoMetaData = struct
     "updateinfo" |> of_xml_file xml_path |> assert_valid
 end
 
+module GroupMetaData = struct
+  include RepoMetaData
```

Comment:
Include adds the fields and methods of `RepoMetaData` into the signature of the module. I don't think this is what should happen, an `open` here is more that enough, or the fully qualified name for fields and functions can be used as well